src/pages/log-management/opensearch/opensearch-dashboards/api.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Access Opensearch Dashboard features via REST API
+metaTitle: Learn How to Access Opensearch Dashboards features via REST API
+description: Read Logit.io’s help article to learn how to access Opensearch Dashboards features via REST API.
+---
+
+# Accessing Opensearch Dashboards features via REST API
+
+You can use Basic Authentication to connect to the Logit.io Proxy and access Opensearch Dashboards without having to go 
+through the Logit.io Dashboard. This can improve your user experience and allow proper integration between your company, your applications 
+and the Logit.io service.
+
+You can, if you choose to, connect your application directly to Logit.io to do a range of things such as load dashboards, visualisations, 
```

Comment:
I don't think the "if you choose to" is necessary but jusy my opinion.

Reply:
I was trying to convey the message that none of this is a must do, but that it can be used if you don't want to keep going through the Logit Dashboard. I'm happy to remove it if we think it is better without.